_pages/en_US/godmode9-usage.txt

```diff
@@ -122,18 +122,52 @@ Insert the game cartridge you intend to dump into your device
   + **NDS Game Cartridge:** Press (A) on `[TitleID].trim.nds` to select it, then select "Copy to 0:/gm9/out"
 1. Your installable `.cia` or non-installable `.nds` formatted file will be outputted to the `/gm9/out/` folder on your SD card
 
-## Dumping a Title
+## Dumping a Game Cartridge's contents
+
+1. Insert a game cartridge you wish to dump from
+1. Press and hold (Start), and while holding (Start), power on your device. This will launch GodMode9
+1. Navigate to `[C:] GAMECART`
+1. Hover over `[TitleID].trim.3ds` and press (A)
+1. Select "NCCH image options"
+1. Select "Mount image to drive"
+1. Press (A) to continue
+1. Hover over the `exefs` directory, hold (R) and press (A) to open the directory options
+1. Select "copy to 0:/gm9/out", do the same with the `romfs` directory
+  + The dumped files will be in `/gm9/out`
+
+## Dumping a Title as a installable .CIA
 
 1. Press and hold (Start), and while holding (Start), power on your device. This will launch GodMode9
 1. Hover over the drive applicable to the type of title you wish to dump:
   + **User Installed Title**: `[A:] SYSNAND SD`
   + **System Title**: `[1:] SYSNAND CTRNAND`
 1. Hold (R) and press (A) at the same time to open the drive options
-1. Select "Search for titles"
+1. Select "Open title manager"
 1. Press (A) to continue
 1. Press (A) on the `.tmd` file to select it, then select "TMD file options...", then select "Build CIA (standard)"
 1. Your installable `.cia` formatted file will be outputted to the `/gm9/out/` folder on your SD card
 
+## Dumping a Title's contents
+
+1. Press and hold (Start), and while holding (Start), power on your device. This will launch GodMode9
+1. Hover over the drive applicable to the type of title you wish to dump the contents of:
+  + **User Installed `Title: [A:] SYSNAND SD`
+  + **System Title: `[1:] SYSNAND CTRNAND`
+1. Hold (R) and press (A) at the same time to open the drive options
+1. Select “Open title manager”
+1. Press (A) to continue
+1. Press (A) on the title you wish to dump the romfs/exefs from 
+  + Any non-alphanumeric characters will be a `?`, like the e in pokemon or Japanese characters
+  + Titles will have the title ID before the name
+1. Press (A) on "Open title folder"
+1. Then press (A) on the biggest file ending in `.app`
+1. Select (A) on "NCCH image options"
+1. Select (A) on "Mount image to drive"
+1. Press (A) to continue
+1. Hover over the `exefs` directory, hold (R) and press (A) to open the directory options
+1. select "copy to 0:/gm9/out". Do the same with the `romfs` directory
+  + The dumped files will be in `/gm9/out`
```

Comment:
I'd reword this to "The dumped files will be located in `/gm9/out` on your SD card" but I'm sure lily would have a better suggestion